restorer.go

```diff
@@ -117,3 +155,72 @@ func (r *Restorer) restoreLayer(cache Cache, sha string) error {
 
 	return layers.Extract(rc, "")
 }
+
+func (r *Restorer) restoreSBOM() error {
+	if api.MustParse(r.Platform.API()).LessThan("0.8") {
```

Comment:
Should we move this gate into the calling function?

Reply:
_Fixed in latest commit._